packages/cli/src/fingerprint/globber.ts

```diff
@@ -14,7 +15,11 @@ class Globber extends EventEmitter {
   public scanNow() {
     assert(!this.currentGlob);
     this.timeout = undefined;
-    this.currentGlob = new Glob(this.pattern, { ignore: ['**/node_modules/**', '**/.git/**'] })
+    this.currentGlob = new Glob(this.pattern, {
+      ignore: ['**/node_modules/**', '**/.git/**'],
+      strict: false,
```

Comment:
`strict` defaults to `false`, so this has no effect.

Reply:
Sorry about that, looks like their doc needs an update: https://github.com/isaacs/node-glob#options .